model/utils_test.go

```diff
@@ -18,30 +18,22 @@ import (
 func TestNewId(t *testing.T) {
 	for i := 0; i < 1000; i++ {
 		id := NewId()
-		if len(id) > 26 {
-			t.Fatal("ids shouldn't be longer than 26 chars")
-		}
+		assert.LessOrEqual(t, len(id), 26, "ids shouldn't be longer than 26 chars")
```

Comment:
the previous test was `len(id) > 26` len(id) should be bigger than 0 and 0 is not included. then I think you cannot use `LessOrEqual` should use `Less` only

Reply:
@cpanato thanks for the feedback! I'm running this test locally with `Less` instead of `LessOrEqual` like `go test -v -run TestNewId` and `len(id)` is coming back ***as*** 26.
```
=== RUN   TestNewId
--- FAIL: TestNewId (0.03s)
    utils_test.go:21: 
        	Error Trace:	utils_test.go:21
        	Error:      	"26" is not less than "26"
        	Test:       	TestNewId
        	Messages:   	ids shouldn't be longer than 26 chars
```
In the previous version if `len(id)` *was* 26 then `len(id) > 26` would resolve to `26 > 26` which is `false`. That means `t.Fatal` never gets called.